app/frontend/components.js

```diff
@@ -39,7 +39,7 @@ window.mountFundraiser = (root: string, store?: Store, initialState?: any = {})
     module.hot.accept('./containers/FundraiserView/FundraiserView', () => {
       const UpdatedFundraiserView = require('./containers/FundraiserView/FundraiserView').default;
       render(
-        <ComponentWrapper store={store}>
+        <ComponentWrapper store={store} locale={'en'}>
```

Comment:
Passing locale prop here, If I understand correctly what this does, this will this break hot reload localization. How can we avoid this? Should we have the locale in a global in JS or maybe store it in the store? Defaulting to ‘en’ for now.